includes/classes/Rest/IssuesAPI.php

```
@@ -0,0 +1,615 @@
+	/**
+	 * Query Data.
+	 *
+	 * @var array
+	 */
+	private $query_data = [];
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		$this->api_version = '1';
```

PHPStan (2.1.15)

Access to an undefined property EqualizeDigital\AccessibilityChecker\Rest\Issues_API::$api_version.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)